test/runtests.jl

```diff
@@ -83,6 +83,35 @@ end
     end
 end
 
+@safetestset "split trilinear interpolation" begin
+    include("split_trilinear_interpolation.jl")
+
+    for FT in (Float32, Float64)
+        for vreverse in (false, true)
```

Comment:
we should also test both options for `ordering`

Reply:
We are testing both orderings, here (https://github.com/CliMA/ClimaInterpolations.jl/pull/16/files#diff-72553a7ff3d5d192daea35e6d03e6a172d2078876209d178e29e716fb92845b2R96) and here (https://github.com/CliMA/ClimaInterpolations.jl/pull/16/files#diff-72553a7ff3d5d192daea35e6d03e6a172d2078876209d178e29e716fb92845b2R110)